tests/receivers/discovery/testdata/host_observer_endpoints_config.yaml

```diff
@@ -1,15 +1,11 @@
 extensions:
   host_observer:
-  host_observer/with_name:
-  host_observer/with/another/name:
```

Comment:
Had to remove these observers because the testing framework doesn't work with several resource metrics having empty resource attributes. We could've fixed the framework, but I'm not sure these extra observers provide any value to the integration test. Also, the framework will be replaced with the `pdatatest` from contrib